uint128/uint128.go

```diff
@@ -0,0 +1,122 @@
+package uint128
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+type Int [2]uint64
+
+func NewInt64(lo uint64) *Int {
```

Comment:
Let's use the `Set...` idiom established by `math/big` and our `core/felt` package.